<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbStockMotion">
     <property name="title">
      <string>Движения</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="gbFilter">
        <property name="title">
         <string>Фильтр</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="1" colspan="2">
          <widget class="CStorageComboBox" name="cmbMotionsFilterSupplier"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblDocType">
           <property name="text">
            <string>Тип документа</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblSource">
           <property name="text">
            <string>Источник</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="CStorageComboBox" name="cmbMotionsFilterReceiver"/>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="CStockMotionTypeComboBox" name="cmbMotionsFilterType"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblDest">
           <property name="text">
            <string>Получатель</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="CDateEdit" name="edtMotionsFilterBegDate"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblPeriod">
           <property name="text">
            <string>Период</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="CDateEdit" name="edtMotionsFilterEndDate"/>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="btnMotionsFilterReset">
           <property name="text">
            <string>Сбросить</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="btnMotionsFilterApply">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CTableView" name="tblMotions"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbStockRequisition">
     <property name="title">
      <string>Требования</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="gbRequisitionFilter">
        <property name="title">
         <string>Фильтр</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="lblRequisitionsPeriod">
           <property name="text">
            <string>Период</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="CStorageComboBox" name="cmbRequisitionsFilterSupplier"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblRequisitionsSource">
           <property name="text">
            <string>Источник</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="CDateEdit" name="edtRequisitionsFilterBegDate"/>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="btnRequisitionsFilterApply">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblRequisitionsDest">
           <property name="text">
            <string>Получатель</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="CStorageComboBox" name="cmbRequisitionsFilterRecipient"/>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="btnRequisitionsFilterReset">
           <property name="text">
            <string>Сбросить</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="CDateEdit" name="edtRequisitionsFilterEndDate"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="CTableView" name="tblRequisitions"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Экспорт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CStockMotionTypeComboBox</class>
   <extends>QComboBox</extends>
   <header>appendix/importNomenclature/StockMotionTypeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ExportDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>647</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
